addons/l10n_us/models/res_bank.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class ResBank(models.Model):
+    _inherit = "res.bank"
+
+    intermediary_bank_id = fields.Many2one(
+        "res.bank",
```

Comment:
while we are at it let's add the comodal_name before res.bank and could be single quotes since it's not something the user can usually see :smile: